src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -29,7 +29,7 @@
 from .util import random_split_by_session as _random_split_by_session
 from .util import _MIN_NUM_SESSIONS_FOR_SPLIT
 
-
+import turicreate.toolkits.libtctensorflow
```

Comment:
We don't want to import TF by default, only if the user has opted into using the TF beta implementation

Reply:
Moved it inside the block where user sets it to `True`